main.c

```diff
@@ -1,13 +1,40 @@
 #include <stdio.h>
+#include <stdlib.h>
+
+struct Node {
+    int data;
+    struct Node* next;
+};
 
 int main() {
-    // TODO: build a linked-list version of this array
-    int numbers[] = {1, 2, 3, 4, 5};
 
-    for (int i = 0; i < 5; i++) {
-        printf("%d\n", numbers[i]);
+    struct Node* head = (struct Node*)malloc(sizeof(struct Node));
```

Comment:
Consider using a loop to create the nodes instead of manually allocating each one. This would make the code more concise and easier to maintain if the size of the list changes.